src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.main {
+  padding: 50px 40px;
+}
```

Comment:
Move to`main.scss`

Reply:
My bad, in `main.scss` should placed only import, rename block to something like `page` and move to other file